src/App.js

```diff
@@ -4,6 +4,7 @@ import "./App.css";
 import { Home } from "./pages/home/home";
 import { ProductPage } from "./pages/productPage/productPage";
 import { CartPage } from "./pages/cartPage/cartPage";
+import { WishlistPage } from "./pages/wishlistPage/wishlistPage";
```

Comment:
Suggestion: you may export all files from `./index.js`

Reply:
thanks for suggestion,i have updated it